UnitySamples/FBLeaderboard/Docs/FBLeaderboardTutorial.md

```diff
@@ -0,0 +1,301 @@
+## Overview
+
+There are many social reasons why a developer would wish to incorporate Facebook into their app (in our case to fetch information about friends in order to create a leaderboard) but one of the main advantages of using Facebook in conjunction with ChilliConnect is that it allows players to link their ChilliConnect account to Facebook.
+
+When a ChilliConnect account is linked to Facebook, players can retrieve their game data across multiple devices or after a fresh reinstall of your app. ChilliConnect provides other means for players to re-access accounts (notably by creating an account with an email address and password) but Facebook is usually more convenient and has an established trust with your users.
+
+This tutorial covers the following:
+
+- Setting up a Facebook App and logging into Facebook using the Unity Facebook SDK.
+- Attaching a Facebook account to a ChilliConnect account so that players can login to ChilliConnect using Facebook.
+- Creating a leaderboard on ChilliConnect
+- Posting scores
+- Retrieving and displaying scores and Facebook profiles for a player's friends.
+
+The full Unity project for this tutorial is available on the ChilliConnect GitHub [samples respository](https://github.com/ChilliConnect/Samples) under the folder "FBLeaderboard". In order to run the project you must configure it with your own ChilliConnect game token and Facebook App Id (see below).
+
+## Creating a Facebook App
+
+As well as having a game configured on the ChilliConnect dashboard you must also create the game (or app) on Facebook. Creating the app will generate the token required to communicate with Facebook and allow us to query which of the player's friends are also playing our game. A more in depth walkthrough about integrating Unity and Facebook can be found [here](https://developers.facebook.com/docs/unity/gettingstarted), this section will cover the most important aspects.
+
+Here are the steps:
+
+- Login to the Facebook account that will be used to create the application (you can add more developers later if you choose).
+- Locate the "Manage Apps" link on the left most navigation bar
+
+TODO: Link to manage apps png
```

Comment:
Mike is going to link these as he needs to upload the images to wherever they are hosted